akka-projection-core/src/main/scala/akka/projection/scaladsl/ProjectionManagement.scala

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Does this implementation only allow users to manage projections that are currently active (registered)?  One use case for the management API might be to validate state of offsets before a projection is started, in which case it would need to work directly with offset stores.

Reply:
We will have functionality to "pause" a projection so I guess that can solve that use case.
That is not part of MVP but covered in ticket https://github.com/akka/akka-projection/issues/25